flow/util/generate-vars.sh

```diff
@@ -46,13 +46,6 @@ while read -r VAR; do
         # skip variables that match the exclude patterns
         continue
     fi
-    # handle special case where the variable needs to be splitted in Tcl code
-    if [[ "${name}" == "GND_NETS_VOLTAGES" || "${name}" == "PWR_NETS_VOLTAGES" ]]; then
-        echo "export ${name}='${value}'" >> $1.sh
-        echo "set env(${name}) ${value}" >> $1.tcl
-        echo "set env ${name} ${value}" >> $1.gdb
-        continue
-    fi
```

Comment:
why removed?

Reply:
It's broken.

Near as I can tell, it was added back in the mists of time by @vvbandeira to solve some problem that I no longer believe exists.

Reply:
With this PR I added to gcd

export PWR_NETS_VOLTAGES  = "VDD1 1.8 VDD2 1.2"

and get

export PWR_NETS_VOLTAGES=""VDD1 1.8 VDD2 1.2""

with master

export PWR_NETS_VOLTAGES='"VDD1 1.8 VDD2 1.2"'

The double-double quotes are not right

PWR_NETS_VOLTAGES=""VDD1 1.8 VDD2 1.2"" tclsh
1.8: command not found

vs

PWR_NETS_VOLTAGES='"VDD1 1.8 VDD2 1.2"' tclsh
% dict for {pwrNetName pwrNetVoltage}  {*}$::env(PWR_NETS_VOLTAGES) { puts $pwrNetName }
VDD1
VDD2

Reply:
Isn't that a bug in the config.mk files, there shouldn't be any quotes there?

No quotes:


```
designs/nangate45/bp_quad/config.mk:export DIE_AREA    = 0 0 3600 3600
designs/nangate45/bp_quad/config.mk:export CORE_AREA   = 10 12 3590 3590 
designs/nangate45/bp_quad/config.mk:export PLACE_PINS_ARGS = -exclude left:* -exclude right:* -exclude top:* -exclude bottom:0-1000 -exclude bottom:2400-3600
```

Reply:
@maliberty Try cherry-picking https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2679 on top of this PR

Reply:
In that case the `{*}` needs to be removed from
```
    dict for {pwrNetName pwrNetVoltage}  {*}$::env(PWR_NETS_VOLTAGES) {
```
(likewise gnd)

Reply:
PWR_NETS_VOLTAGES="VDD1 1.8 VDD2 1.2" tclsh
% dict for {pwrNetName pwrNetVoltage}  {*}$::env(PWR_NETS_VOLTAGES) { puts $pwrNetName}
wrong # args: should be "dict for {keyVarName valueVarName} dictionary script"

vs

dict for {pwrNetName pwrNetVoltage}  $::env(PWR_NETS_VOLTAGES) { puts $pwrNetName}
VDD1
VDD2

Reply:
You obviously know what is going on here, could you clean out the inconsistent & superfluous quotes + fix the .tcl?